pages/api/exportStats.ts

```diff
@@ -0,0 +1,56 @@
+import { withApiAuth } from "../../lib/auth/withApiAuth";
+import { withApiMethods } from "../../lib/withApiMethods";
+import { prisma } from "../../lib/db";
+const excel = require("excel4node");
```

Comment:
Too much [Require Podcast](https://require.podcast.gq)?

Reply:
![image](https://user-images.githubusercontent.com/35652893/145727006-749ffe24-6a5d-42b2-9932-082c8c91ad6f.png)

Reply:
yes, es6 import is indeed joke to me 😈